imageBuilder.go

```diff
@@ -11,6 +12,21 @@ func main() {
 	// Unmarshal the yaml configuration file and generate a dockerfile.
 	err := configurations.WriteDockerfile()
 	if err != nil {
-		log.Fatalf("error while writing dockerfile %v", err)
+		log.Fatalf("error while writing dockerfile: %v", err)
+	}
+
+	authData, err := builder.GetAuthData("docker-auth.yaml")
```

Comment:
magic string!! Use environment variables for dockerfile name, docker auth file name and so on.

Reply:
Okay